src/rpc/rawtransaction.cpp

```diff
@@ -1541,7 +1541,7 @@ static RPCHelpMan finalizepsbt()
     std::string result_str;
 
     if (complete && extract) {
-        ssTx << mtx;
+        ssTx << TX_WITH_WITNESS(mtx);
```

Comment:
These can be a `DataStream`s now?

Reply:
None of the psbt bits can switch to `DataStream` until `CVectorWriter` is updated.